### DISCUSSION TITLE
Turbulence sound effects

### DISCUSSION THREAD

**Discussion by studio051** [flybywiresim/aircraft#4815] [Ideas]
Just a suggestion , having turbulence sounds in the cockpit like squeaks and rattles would really add to the immersion . Options can be enabled or disabled in the EFB as per preference . Just a suggestion.

**Comment by hotshotp** [comment]
this is being worked on by me, its just hard to get a balance of too much shaking at too little turbulence or vice versa, that kind of stuff.

**Reply by studio051** [reply]
Great , good luck .

**Comment by scoopscode** [comment]
_hears cabin flying around in the heavy turbulence_
